app/services/api/auth/login.js

```diff
@@ -1,43 +1,37 @@
 angular.module('login.service', [])
     .factory('loginService',
-    ['$q', '$timeout', '$http', 'dashboardService',
-        function ($q, $timeout, $http, dashboardService) {
+    ['$q', 'credentials',
+        function ($q, credentials) {
```

Comment:
Done the changes. 
The credentials dependency is defined as a provider in app file. It was mentioned in the assignment - 3. The loginService will use a 'credentials' dependency (created as  a provider and will store all credentials) and will search for any combination of username/password in the list to validate